app/models/scoreboard.rb

```diff
@@ -45,7 +45,7 @@ def active_academic_years(today: Time.zone.today)
     calendar_area.academic_years.where("date_part('year', start_date) >= ? AND start_date <= ?", FIRST_YEAR, today).order(:start_date)
   end
 
-  def scored_schools(recent_boundary: 1.month.ago, academic_year: nil)
+  def scored_schools(recent_boundary: 1.month.ago, academic_year: this_academic_year)
```

Comment:
Scoreboards were updating properly, but the other places with points, like the Podium still had last year's points totals..

Hope I did the right things ..